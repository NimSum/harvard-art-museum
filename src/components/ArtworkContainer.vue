<template>
  <section class="ArtworkContainer"> 
      <ArtworkItem 
        v-for="artwork in artworks"
        v-bind:key="artwork.objectid"
        v-bind:artwork="artwork" 
      />
  </section>
</template>

<script>
import ArtworkItem from './ArtworkItem';

export default {
  name: 'ArtworkContainer',
  props: ['artworks'],
  components: {
    ArtworkItem
  }
}
</script>

<style lang="scss" scoped>
  .ArtworkContainer {
    width: 80%;
    height: 100vh;
    display: flex;
    flex-wrap: wrap;
    overflow: scroll;
    padding: 1%;
    background-color: #E3FDFD;
  }

</style>

