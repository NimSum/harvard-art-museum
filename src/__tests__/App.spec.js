import { mount } from '@vue/test-utils';
import ArtworkContainer from '../components/ArtworkContainer.vue';

describe('ArtworkContainer', () => {
  test('renders correctly', () => {
    const wrapper = mount(ArtworkContainer);
    expect(wrapper.element).toMatchSnapshot();
  })
});