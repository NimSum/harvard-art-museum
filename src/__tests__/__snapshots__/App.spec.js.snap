// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<div
  id="app"
>
  <header>
    <router-link
      to="/"
    >
      <h1>
        <span>
          Harvard Art
        </span>
        Musem Explorer
      </h1>
    </router-link>
     
    <ul>
      <li>
        <router-link
          to="/"
        >
          
          Home
      
        </router-link>
      </li>
       
      <li>
        <router-link
          to="/floor/1"
        >
          
         Floor
          <span>
            1
          </span>
        </router-link>
      </li>
       
      <li>
        <router-link
          to="/floor/2"
        >
          
        Floor
          <span>
            2
          </span>
        </router-link>
      </li>
       
      <li>
        <router-link
          to="/floor/3"
        >
          
        Floor
          <span>
            3
          </span>
        </router-link>
      </li>
       
      <li>
        <router-link
          to="/floor/4"
        >
          
        Floor
          <span>
            4
          </span>
        </router-link>
      </li>
    </ul>
  </header>
   
  <router-view />
</div>
`;
